torchci/lib/bot/verifyDisableTestIssueBot.ts

```diff
@@ -10,17 +12,17 @@ export const disabledTestIssueTitle = new RegExp(
 );
 export const pytorchBotId = 54816060;
 
-export const supportedPlatforms = new Set([
-  "asan",
-  "linux",
-  "mac",
-  "macos",
-  "rocm",
-  "slow",
-  "win",
-  "windows",
-  "dynamo",
-  "inductor",
+export const supportedPlatforms = new Map([
+  ["asan", []],
+  ["linux", []],
+  ["mac", ["module: macos"]],
+  ["macos", ["module: macos"]],
+  ["rocm", ["module: rocm"]],
+  ["slow", []],
+  ["win", ["module: windows"]],
+  ["windows", ["module: windows"]],
+  ["dynamo", ["oncall: pt2"]],
+  ["inductor", ["oncall: pt2"]],
```

Comment:
the processes later will add "triaged" for the "module" ones right?

Reply:
I think `triaged` label could all be added for `module` and `oncall` right here now that it's a list of labels

Reply:
No need to add triaged for oncall labels btw